From 93a9db056aadca1753752869ec7bfa767e1571b6 Mon Sep 17 00:00:00 2001
From: Craig Tatlor <ctatlor97@gmail.com>
Date: Mon, 21 May 2018 17:31:58 +0100
Subject: [PATCH 1/5] hal: Build with BOARD_VNDK_VERSION

Adds includes that arent included by default with BOARD_VNDK_VERSION.

Change-Id: I6e785bfaab77bd39c5dea22daaa5d6cb28fae403
---
 hal/audio_extn/audio_extn.c | 1 +
 hal/audio_hw.c              | 3 +++
 hal/msm8974/platform.c      | 1 +
 hal/voice_extn/voice_extn.c | 1 +
 4 files changed, 6 insertions(+)

diff --git a/hal/audio_extn/audio_extn.c b/hal/audio_extn/audio_extn.c
index 6a7dde46e..9eeb5e3f7 100644
--- a/hal/audio_extn/audio_extn.c
+++ b/hal/audio_extn/audio_extn.c
@@ -23,6 +23,7 @@
 
 #include <stdlib.h>
 #include <errno.h>
+#include <unistd.h>
 #include <dlfcn.h>
 #include <cutils/properties.h>
 #include <cutils/log.h>
diff --git a/hal/audio_hw.c b/hal/audio_hw.c
index 6fd8d6bc1..0d1306d3b 100644
--- a/hal/audio_hw.c
+++ b/hal/audio_hw.c
@@ -29,11 +29,14 @@
 #include <pthread.h>
 #include <stdint.h>
 #include <sys/time.h>
+#include <time.h>
 #include <stdlib.h>
 #include <math.h>
+#include <unistd.h>
 #include <dlfcn.h>
 #include <sys/resource.h>
 #include <sys/prctl.h>
+#include <sys/ioctl.h>
 
 #include <cutils/log.h>
 #include <cutils/str_parms.h>
diff --git a/hal/msm8974/platform.c b/hal/msm8974/platform.c
index 526e0aca6..ca96c6999 100644
--- a/hal/msm8974/platform.c
+++ b/hal/msm8974/platform.c
@@ -23,6 +23,7 @@
 
 #include <stdlib.h>
 #include <dlfcn.h>
+#include <unistd.h>
 #include <fcntl.h>
 #include <cutils/log.h>
 #include <sys/ioctl.h>
diff --git a/hal/voice_extn/voice_extn.c b/hal/voice_extn/voice_extn.c
index acbf861a2..6e100c5bd 100644
--- a/hal/voice_extn/voice_extn.c
+++ b/hal/voice_extn/voice_extn.c
@@ -23,6 +23,7 @@
 
 #include <errno.h>
 #include <math.h>
+#include <time.h>
 #include <cutils/log.h>
 #include <cutils/str_parms.h>
 #include <sys/ioctl.h>
-- 
2.17.0

